package-lock.json

```diff
@@ -1262,9 +1262,10 @@
     },
     "node_modules/@webgpu/types": {
       "version": "0.1.27",
-      "resolved": "https://registry.npmjs.org/@webgpu/types/-/types-0.1.27.tgz",
+      "resolved": "git+ssh://git@github.com/gpuweb/types.git#48dafa2124c767bcc4f84b0d1126bcc478143fad",
```

Comment:
No matter what format I use in package.json, even explicitly specifying `https://`, this always ends up resolving to `git+ssh://`. This seems like it could cause issues if ssh is not set up on a machine, but it seems to pass the build check, so hopefully it's fine. We can revert back to the other format if not.